ui/app/registration/controllers/editPatientController.js

```diff
@@ -98,11 +98,8 @@ angular.module('bahmni.registration')
             };
 
             $scope.sendWhatsAppMessage = function () {
```

Comment:
can we re-name this function

Reply:
renamed the function to notifyOnWhatsApp